### PR TITLE
Added italian translation file

### PR DESCRIPTION
**Pull request by joethefox**
Hi, very neat plugin :)

this PR adds the italian translation to the project

Regards

### CODE REVIEW

File: config/locales/it.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    close: "Chiudi"
+    comment: "Commento"
+    commit: "Fine registrazione"
+    discard: "Annulla"
+    logging_since: "Regitrando tempo su %{descr} da %{time}"
+    need_rest_api: "Per favore verificare che 'Abilita il servizio web REST' è\n abilitato in Amministrazione Redmine/Impostazioni/API"
+    no_permission: "Non si dispone delle autorizzazioni necessarie per segnare il tempo al progetto %{project}."
+    nothing_inferred: "Nessun progetto o segnalazione può essere dedotta dalla pagina corrente."
+    notification_title: "Segnate %{hours} ore a %{descr}"
+    plugin_name: "Bestest Punch Clock for Redmine"
```
Comment (LeviticusMB) [inline config/locales/it.yml]: Would you mind if I asked you to translate the plugin name as well? 

("Bestest" i a childish word for the *very best* or *better than best* and "Punch Clock" is [one of these](https://duckduckgo.com/?q=punch+clock&t=ffsb&iax=1&ia=images), of course ...)

**Comment by LeviticusMB** [comment]
I've also added a language key that was missing:  `submit_failed`.

File: config/locales/it.yml (hunk 1 above)
Comment (joethefox) [inline config/locales/it.yml]: Sure, I didn't translated it thinking about, wrongly maybe, that a name is a name and it should not be translated. Such as Redmine is Redmine, not Rossomio in italian :D. 
Translation is arriving...

**Comment by LeviticusMB** [comment]
Awesome. Thanks!

**Pull request merged**